src/scripts/content/gemini.js

```diff
@@ -3,18 +3,20 @@
 'use strict';
 
 togglbutton.render('#filter-navigator-container:not(.toggl)', {observe: true}, function () {
-  var link, description,
+  var link, description, id,
     numElem = $('.item-title'),
     titleElem = $('.item-title'),
     projectElem = $('.project-info a');
 
   description = titleElem.textContent;
   if (numElem !== null) {
-    description = numElem.textContent + " " + description;
+    id = numElem.textContent.split(" ")[0];
+    description = description.split(" ").splice(1);
```

Comment:
I tested all of them and they worked, if they didn't I'd not push. I could try to fix it as you suggested, but my account on Gemini has expired, so I wouldn't be able to test :(